Exporter/src/test/java/uk/ac/ebi/biostd/exporter/test/FtpServiceTest.java

```diff
@@ -57,16 +59,46 @@ public void generateFtpLinks() {
 
         assertTrue(Files.exists(Paths.get(
             folder.getRoot().getAbsolutePath() + "/ftp/S-EPMC/S-EPMCxxx633/S-EPMC3343633/File1.txt")));
+        assertTrue(Files.exists(Paths.get(
+            folder.getRoot().getAbsolutePath() + "/ftp/S-EPMC/S-EPMCxxx634/S-EPMC3343634/File2.txt")));
+        assertFalse(Files.exists(Paths.get(
+            folder.getRoot().getAbsolutePath() + "/ftp/S-EPMC/S-EPMCxxx633/S-EPMC3343633/OutdatedFile1.txt")));
+    }
+
+    @Test
+    public void generateFtpLinksByAccNo() {
+        testInstance.execute("S-EPMC3343634");
+
+        assertFalse(Files.exists(Paths.get(
+            folder.getRoot().getAbsolutePath() + "/ftp/S-EPMC/S-EPMCxxx633/S-EPMC3343633/File1.txt")));
```

Comment:
The looks kind of verbose not  we may create utility methods as  `assertFtpFileExist("S-EPMC3343633/File1.txt")`